src/base/utils/foreignapps.cpp

```diff
@@ -29,11 +29,19 @@
 
 #include "foreignapps.h"
 
+#if defined(Q_OS_WIN)
+#include <windows.h>
```

Comment:
`winreg.h` as in the old code?

Reply:
Ran into this error:
```
C:\Program Files (x86)\Windows Kits\10\include\10.0.14393.0\um\winnt.h(154): fatal error C1189: #error:  "No Target Architecture"
```
IIRC results from goolgling suggest include windows.h instead.

Reply:
OK, but its weird that the old code compiled with that header and the new one does not.

Reply:
Hmm, it seems that `shlobj.h` included `windows.h` that's why `winreg.h` worked. (it actually didn't work, the sub-included windows.h did all the work)